### ISSUE TITLE
"namespace is too long" 

### ISSUE THREAD

**Issue by stefanve** [matteodem/meteor-easy-search#642]
MongoError: namespace name generated from index name "5c50994dd5ec13fbd291e333_db.coll.$field1_text__id_text_field2_text_field3_text_field4_text" is too long (127 byte max)

Only way I can get around that error is to remove some fields from the engine definition.

**Comment by matteodem** [comment]
Are you using the mongo text index engine?

**Comment by stefanve** [comment]
> Are you using the mongo text index engine?

Yes. Or I did. I got around the problem by simply pre-creating the indexes with more handsome names.
